examples/raytracing-showcase/processors/split_screen_compositor.py

```
@@ -0,0 +1,329 @@
+"""
+
+from __future__ import annotations
+
+import struct
+
+from streamlib import (  # noqa: A004 — `input` is streamlib's port decorator
+    ProcessorOutputTextureRing,
+    RuntimeContextFullAccess,
+    RuntimeContextLimitedAccess,
+    VideoFrame,
+    input,
```

Ruff (0.16.2)

[error] Import `input` is shadowing a Python builtin

(A004)